### ISSUE TITLE
Large files in repo

### ISSUE THREAD

**Issue by Jhsmit** [MarcSkovMadsen/awesome-analytics-apps-template#2]
I tried to use the template functionality to 'fork' the repo but this seems to not be possible with large files present:

![image](https://user-images.githubusercontent.com/7881506/103890177-e71c6f80-50e7-11eb-87b2-2e4bc174a52e.png)

I'll download the files instead as you describe in the readme meanwhile.

**Comment by MarcSkovMadsen** [comment]
Thanks @Jhsmit . The large files are a few images and a video used in some of the pages. I guess I should serve them from somewhere else.

Did you manage to get it downloaded and running?

**Comment by Jhsmit** [comment]
Yes, I had not trouble getting it to run out of the box. (Firefox or Chrome both work fine)

Now it's time to break it 😈
